serde/se.py

```diff
@@ -507,15 +526,21 @@ def render(self, arg: SeField) -> str:
             IPv4Interface,
             IPv6Interface,
         ]:
-            return f"{arg.varname} if reuse_instances else {self.string(arg)}"
+            res = f"{arg.varname} if reuse_instances else {self.string(arg)}"
         elif arg.type in [date, datetime]:
-            return f"{arg.varname} if reuse_instances else {arg.varname}.isoformat()"
+            res = f"{arg.varname} if reuse_instances else {arg.varname}.isoformat()"
         elif is_none(arg.type):
-            return "None"
+            res = "None"
         elif arg.type is Any:
-            return f"to_obj({arg.varname}, True, False, False)"
+            res = f"to_obj({arg.varname}, True, False, False)"
+        else:
+            res = f"raise_unsupported_type({arg.varname})"
+
+        # Custom field serializer overrides custom class serializer.
+        if self.custom and not arg.serialize:
+            return f'serde_custom_class_serializer({arg.type.__name__}, {arg.varname}, default={res})'
```

Comment:
For serialize the default value should probably also be a lambda.

Reply:
True. I will take a look.